addons/account/models/account_bank_statement.py

```diff
@@ -152,7 +152,7 @@ def _compute_ending_balance(self):
                     statement.balance_end_real = statement.previous_statement_id.balance_end_real + total_entry_encoding
             else:
                 # Need default value
-                statement.balance_end_real = statement.balance_end_real or 0.0
+                statement.balance_end_real = statement.balance_end or 0.0               
```

Comment:
I don't think this is the right fix for it. 'balance_end' and 'balance_end_real' should not be mixed. I think that `_check_pos_session_balance` is the right method to do what you want. Can you check and, if I'm wrong and you are sure of your fix, explain a bit more ?

Also, your fix breaks the tests (see runbot).